FileSystem.sln

```diff
@@ -11,23 +11,23 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		NuGet.config = NuGet.config
 	EndProjectSection
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.FileProviders.Abstractions", "src\Microsoft.Extensions.FileProviders.Abstractions\Microsoft.Extensions.FileProviders.Abstractions.csproj", "{DD94B7E8-3A59-4F84-98A0-8139BE259A87}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.FileProviders.Abstractions", "src\Abstractions\Abstractions.csproj", "{DD94B7E8-3A59-4F84-98A0-8139BE259A87}"
```

Comment:
Per the other discussions these should probably all be `FS.Abstractions`, `FS.*` etc. Otherwise you end up with duplicate project filenames that are difficult to disambiguate.